src/assets/scully-routes.json

```
@@ -0,0 +1 @@
+[{"route":"/lup"},{"route":"/wsp"},{"route":"/flp"},{"route":"/faq"},{"route":"/contact"},{"route":"/compliance-oversight"}]
```

We should also prerender the home page (`/`), unless there's dynamic content there I'm not seeing
```suggestion
[{"route":"/"},{"route":"/lup"},{"route":"/wsp"},{"route":"/flp"},{"route":"/faq"},{"route":"/contact"},{"route":"/compliance-oversight"}]
```

---

As discussed, it's unfortunately dynamic right now. But we should have a ticket for removing the updates section and converting it to static, pending Steve's approval.